examples/restore_login.py

```diff
@@ -1,71 +1,90 @@
+#!/usr/bin/env python3
+
 import asyncio
 import json
 import os
 import sys
+import getpass
 
 from nio import AsyncClient, LoginResponse
 
 CONFIG_FILE = "restore_login_config.json"
 
 # Check out main() below to see how it's done.
 
-def write_details_to_disk(resp: LoginResponse) -> None:
+
+def write_details_to_disk(resp: LoginResponse, home_server) -> None:
```

Comment:
I think `home_server` should be one word ("homeserver" is a complete word) and should have a type here. We're going for Best Practices in this example.

Reply:
I agree with you. I had homeserver originally. Then in some other source code I found home_server, so I thought that I adapt to what I have seen. 

Anyway, commit 2 will have home_server replaced with homeserver as you suggested.